integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -675,12 +675,18 @@ public void ttlRenameTest() throws Exception {
     CreateOptions options = new CreateOptions.Builder(MasterContext.getConf()).setTTL(ttl).build();
     long fileId = mFsMaster.create(new TachyonURI("/testFolder/testFile1"), options);
     mFsMaster.renameInternal(fileId, new TachyonURI("/testFolder/testFile2"), true,
-        System.currentTimeMillis());
+        getFakeTimeMs());
     FileInfo folderInfo =
         mFsMaster.getFileInfo(mFsMaster.getFileId(new TachyonURI("/testFolder/testFile2")));
     Assert.assertEquals(ttl, folderInfo.ttl);
   }
 
+  private long getFakeTimeMs() {
+    // Modify current time so that implementations can't accidentally pass unit tests by ignoring
+    // this specified time and always using System.currentTimeMillis()
+    return System.currentTimeMillis() - 1000;
```

Comment:
Is it possible to return a constant as the fake time? In that case, unit tests are more deterministic.